blocks/single-chain-block/chains/single-chain/default.test.jsx

```diff
@@ -16,7 +16,9 @@ describe("single chain", () => {
 
 			expect(wrapper.text()).toBe(testText);
 			expect(wrapper.find("HeadingSection").exists()).toBe(false);
-			expect(wrapper.html()).toBe(`<p>${testText}</p>`);
+			expect(wrapper.html()).toBe(
+				`<div class="c-stack b-single-chain" data-style-direction="vertical" data-style-justification="start" data-style-alignment="unset" data-style-inline="false" data-style-wrap="nowrap"><div class="c-stack b-single-chain__children-stack" data-style-direction="vertical" data-style-justification="start" data-style-alignment="unset" data-style-inline="false" data-style-wrap="nowrap"><p>one chainz</p></div></div>`
+			);
```

Comment:
I think this can be deleted, it's not adding any value and will break if the stack changes.  We don't care that it's in a stack inside another stack, do we?  Just the `text()` above should be sufficient.

Reply:
I removed it